doc/source/changes/version_0_32.rst.inc

```diff
@@ -10,7 +10,9 @@ Syntax changes
 Backward incompatible changes
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-* other backward incompatible changes
+* Dumping and loading Axis and Group objects contained in a session is no longer possible
+  with the CSV and Excel formats. Please use the HDF format instead.
```

Comment:
please explain briefly why this was done. Something vaguely like "because that feature was broken and fixing it was not worth it because it was unused anyway (as far as we know)"